docs/build-&-deployment.md

```diff
@@ -103,62 +93,101 @@ export DEBUG_KEYSTORE_PASSWORD=android
 
 #### Step 1:
 
-Install Chocolatey, refer https://chocolatey.org/install
-```
-Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
-```
-Allow global confirmation to avoid providing confirmations for avoiding confirmation in each installation (Optional)
+Install Git
+
+Use the below link to download git
 ```
-choco feature enable -n=allowGlobalConfirmation
+https://git-scm.com/download/win
 ```
+After installing run Git as admin
 
 #### Step 2:
 
-Configure Git
+Install SDKMAN
+
+Use the below command in Git terminal
+
 ```
-choco install git
+curl -s "<https://get.sdkman.io>" | bash
 ```
 
+If encounters, `please install zip on your system using your favourite package manager` error while installing sdkman
```

Comment:
If encounters, `please install zip on your system using your favourite package manager` error while installing sdkman-- Sentence not clear... Please rephrase...